topics/kernel/index.md

```diff
@@ -0,0 +1,9 @@
+---
+aliases: monolithic, microkernel, micro-kernel, exokernel, nanokernel
```

Comment:
`monolithic` could mean different things in different contexts, so I'm not sure it's an appropriate alias.

```suggestion
aliases: microkernel, micro-kernel, exokernel, nanokernel
```